topic-controller/src/test/resources/ControllerIT-rbac.yaml

```diff
@@ -0,0 +1,46 @@
+---
+# Extra permissions required by the integration tests
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: strimzi-topic-controller-role-it
+  labels:
+    app: strimzi
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - list
+  - watch
+  - get
```

Comment:
Does the topic controller needs to "create" events ? Does it need to list/watch/get ?

Reply:
The TC creates events when something goes wrong, yes.

The `ControllerIT` then makes assertions about those events, hence the need to list+get. TBH watch is probably not needed. Let me fix that.